Doc/source/designspaceLib/readme.rst

```diff
@@ -312,16 +309,16 @@ RuleDescriptor object
 
 -  ``name``: string. Unique name for this rule. Will be used to
```

Comment:
here we should mention that the rule "name" is optional: 

https://github.com/fonttools/fonttools/issues/1243#issuecomment-385252995

> The name attribute of the rule element should be optional. It's not needed when compiling a font.